<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>打印页面</string>
     </property>
     <addaction name="actionExImage"/>
     <addaction name="separator"/>
     <addaction name="actionExPDF"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="menu_2"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>配置</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>主题</string>
     </property>
     <addaction name="actionLight"/>
     <addaction name="actionDark"/>
    </widget>
    <widget class="QMenu" name="menuGrid">
     <property name="title">
      <string>栅格大小</string>
     </property>
     <addaction name="action1_pixel"/>
     <addaction name="separator"/>
     <addaction name="action2_pixel"/>
     <addaction name="separator"/>
     <addaction name="action5_pixel"/>
     <addaction name="separator"/>
     <addaction name="action10_pixel"/>
    </widget>
    <addaction name="menuTheme"/>
    <addaction name="separator"/>
    <addaction name="menuGrid"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLine">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEllipse">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/images/New.png</normaloff>:/Resources/images/New.png</iconset>
   </property>
   <property name="text">
    <string>新建页面</string>
   </property>
   <property name="toolTip">
    <string>新建一张图纸</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/images/Open.png</normaloff>:/Resources/images/Open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开页面</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/images/Save.png</normaloff>:/Resources/images/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存页面</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>明亮</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>暗黑</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>保存为</string>
   </property>
  </action>
  <action name="action1_pixel">
   <property name="text">
    <string>1 pixel</string>
   </property>
  </action>
  <action name="action2_pixel">
   <property name="text">
    <string>2 pixel</string>
   </property>
  </action>
  <action name="action5_pixel">
   <property name="text">
    <string>5 pixel</string>
   </property>
  </action>
  <action name="action10_pixel">
   <property name="text">
    <string>10 pixel</string>
   </property>
  </action>
  <action name="actionExImage">
   <property name="text">
    <string>导出图片</string>
   </property>
  </action>
  <action name="actionExPDF">
   <property name="text">
    <string>导出PDF</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
